blocks/core/ff_container/ff_container-expandable/ff_container-expandable.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+$(function(){ 
+	$("[data-ff-action='expandable']").click(function(){
+		var $expandableContent = $("[data-ff='expandable-content']"),
+		$expandableIcon = $("[data-icon='expandable-icon']"),
+		$expandableText = $("[data-ff='expandable-text']");
+		
+		var $expandableOpenText = $expandableText.attr("data-open-text"),
+		$expandableClosedText = $expandableText.attr("data-closed-text");
+
+		var isVisible = $expandableContent.is(":visible");
+		var newText = isVisible ? $expandableClosedText : $expandableOpenText;
+
+		if (isVisible) {
+			$expandableContent.hide();
+			$expandableIcon.removeClass("ff_icon-page-up-open-blue").addClass("ff_icon-page-down-open-blue");
```

Comment:
Do we want to make this a bit more generic? I'm thinking of other themes that wouldn't use the blue icons. 

Hmm, but how to do this? Good question Jon, I'm glad I asked!

Well, first stage would be to pass the icons through in a data attr, maybe `data-ff-icon-closed="ff_icon-page-down-open-blue"`

Second, is how the theme could pass that through to the pattern. Have you any ideas on this @dkhuntrods / @mwilliamson-firefly ?